crates/stackable-certs/src/ca/mod.rs

```diff
@@ -468,19 +498,57 @@ fn format_leaf_certificate_subject(name: &str, scope: &str) -> Result<Name> {
 
 #[cfg(test)]
 mod tests {
+    use const_oid::ObjectIdentifier;
+
     use super::*;
 
+    const TEST_CERT_LIFETIME: Duration = Duration::from_hours_unchecked(1);
+    const TEST_SAN: &str = "airflow-0.airflow.default.svc.cluster.local";
```

Comment:
question: Can we use a non-product specific test value here? This comes to mind:

```suggestion
    const TEST_SAN: &str = "example-0.example.default.svc.cluster.local";
```

Reply:
I personally prefer having "real" names for easier understanding.
The people reading this are developers of our operators and I think concepts are easier to understand with actual, recognizable values

Reply:
> The people reading this are developers of our operators and I think concepts are easier to understand with actual, recognizable values

They might be, this might also get used outside of Stackable and we should strive to make all of our crates usable outside of Stackable. There might not be many external users _now_, but there hopefully are a bunch in the future.

I won't block this PR on this, but we should definitely revisit this in the future if we leave it as is.

Reply:
I also think we should use example placeholders instead of a particular product name (same goes with test versions unless behavior changes of course).

But perhaps instead of `example`, it can be `product` or `product-name`?

```suggestion
    const TEST_SAN: &str = "product-0.product.default.svc.cluster.local";
```